mypyc/irbuild/specialize.py

```diff
@@ -185,6 +185,33 @@ def translate_set_from_generator_call(
     return None
 
 
+@specialize_function('builtins.min')
+def faster_min(builder: IRBuilder, expr: CallExpr, callee: RefExpr) -> Optional[Value]:
+    if (len(expr.args) > 0
```

Comment:
I think that this check is redundant (and not sufficient, since one argument is not enough), since if `arg_kinds` has two positional items, there will be exactly two arguments.